vms/volunteer/services.py

```diff
@@ -58,28 +70,38 @@ def get_volunteer_resume_file_url(volunteer_id):
     if volunteer and volunteer.resume_file:
         result = volunteer.resume_file.url
 
-    return result 
+    return result
+
 
 def get_volunteers_ordered_by_first_name():
     volunteer_list = Volunteer.objects.all().order_by('first_name')
     return volunteer_list
 
+
 def has_resume_file(volunteer_id):
-    
-    result = False 
+
+    result = False
     volunteer = get_volunteer_by_id(volunteer_id)
 
     if volunteer and volunteer.resume_file:
-        result = True 
+        result = True
 
     return result
 
-def search_volunteers(first_name, last_name, city, state, country, organization):
-    
-    #if no search parameters are given, it returns all volunteers
+
+def search_volunteers(
```

Comment:
The search function should reflect that none, one, or more parameters may be sent. Docstring for function needed. Also, I'm not sure if there should be default values set. Thoughts?